src/Resources/config/debug.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+use Kreyu\Bundle\DataTableBundle\DataCollector\DataTableDataCollector;
+use Kreyu\Bundle\DataTableBundle\Debug\TraceableDataTableFactory;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('debug.kreyu_data_table.factory', TraceableDataTableFactory::class)
+        ->decorate('kreyu_data_table.factory')
+        ->args([
+            service('.inner'),
+            service('serializer.data_collector'),
+        ])
+
+        ->set('serializer.data_collector', DataTableDataCollector::class)
```

Comment:
This service should be named `kreyu_data_table.debug.data_collector`, so it is consistent with other services, and to prevent overriding the `serializer.data_collector` provided by the Symfony itself. In my test application, this was causing an error, as it broke Symfony services that depend in this collector:

> Symfony\Component\Serializer\Debug\TraceableNormalizer::__construct(): Argument #2 ($dataCollector) must be of type Symfony\Component\Serializer\DataCollector\SerializerDataCollector, Kreyu\Bundle\DataTableBundle\DataCollector\DataTableDataCollector given, called in /application/var/cache/dev/ContainerId1NLFK/App_KernelDevDebugContainer.php on line 2501